### PR TITLE
Extend mix version check to trigger on 1.3.[0-9]

### PR DESCRIPTION
**Pull request by BartOtten**

### CODE REVIEW

**Comment by BartOtten** [comment]
I think we should be a bit smarter about this. Potentially, every mix release will break the tool and cause headache to dumb devs like me.

**Comment by brienw** [comment]
I misread Jose's comment in https://github.com/elixir-lang/elixir/issues/5094#issuecomment-252489258 when I fixed this last so i limited it to 1.3.[0123] at that time. We should just expand this match to just catch all 1.3.* versions

**Comment by philipgiuliani** [comment]
I think it should be fine when its changed to `1.3.[0-9]`.

File: libexec/erlang [hunk 1]
```diff
@@ -166,10 +166,10 @@ erlang_clean_compile() {
       [[ \"$SKIP_MIX_CLEAN\" != \"true\" ]] && $REBAR_CMD clean skip_deps=true || :
       $REBAR_CMD compile
     elif [ \"$BUILD_CMD\" = \"mix\" ] && [ \"$RELEASE_CMD\" = \"mix\" ]; then
-      echo \"Checking whether deps must be compiled for mix version 1.3.[0123]\"
+      echo \"Checking whether deps must be compiled for mix version 1.3.[01234]\"
       # see https://github.com/boldpoker/edeliver/issues/94
-      if $MIX_CMD --version | grep 'Mix 1.3.[0123]' >/dev/null 2>&1 ; then
-        echo \"Compiling deps because mix version 1.3.[0123] is used\"
+      if $MIX_CMD --version | grep 'Mix 1.3.[01234]' >/dev/null 2>&1 ; then
+        echo \"Compiling deps because mix version 1.3.[01234] is used\"
```
Comment (brienw) [inline libexec/erlang]: Let's just catch all 1.3 versions with `grep 'Mix 1.3.'`

**Comment by BartOtten** [comment]
Should do now.

**Comment by brienw** [comment]
As per my code review comment from a few weeks ago, this still needs to be opened up to all 1.3 versions. please change the check to `grep 'Mix 1.3.'` to catch them all

**Comment by brienw** [comment]
please close this one in favor of the correct check submitted in PR #178

**Comment by lessless** [comment]
Thanksm, @brienw. Closing now.

**Pull request closed without merging**